ui/app/routes/jobs/run/index.js

```diff
@@ -33,29 +34,25 @@ export default class JobsRunIndexRoute extends Route {
       // available namespaces to prevent redirecting to a non-existent namespace.
       await this.store.findAll('namespace');
 
-      // If template is set in URL, create the job model and add the definition
       if (template) {
-        const templateRecord = await this.store.findRecord(
-          'variable',
-          template
-        );
-
+        const VariableAdapter = this.store.adapterFor('variable');
+        const templateRecord = await VariableAdapter.getJobTemplate(template);
         return this.store.createRecord('job', {
           _newDefinition: templateRecord.items.template,
         });
+      } else {
+        return this.store.createRecord('job');
```

Comment:
`nit` We don't need an `else` block here because we return out of the `if` statement.